tests/integration/test_active_response/test_execd/test_execd_firewall_drop.py

```diff
@@ -279,6 +285,9 @@ def test_execd_firewall_drop(set_debug_mode, get_configuration, test_version, co
     tags:
         - simulator
     '''
+    # Check if the agent is restarted properly"
+    assert not start_agent
```

Comment:
Include a message to this assert in order to clarify produced report. Something like the following will do the job
```suggestion
    # Check if the agent is restarted properly"
    assert not start_agent, 'The agent failed to restart successfully after enrolling the authentication simulator.'
```

Reply:
Change done.